examples/precomp/foo/Foo/Foo.lean

```diff
@@ -0,0 +1,2 @@
+def foo := "foo"
+--#eval IO.sleep 3000
```

Comment:
What is this comment for? Also, what exactly does this whole example test? There doesn't seem to be anything that checks whether things are working right?

Reply:
The whole `Foo/` folder is simply copied from `examples/deps/`, but I can change the content. The important parts that are tested are compiling and loading the shared libraries, if that works we're probably good. I'll add an `#eval` of an imported constant though to check that evaluation won't crash and burn.

Reply:
Sounds good!

Reply:
Ooh, it crashed and burned

Reply:
Ah, sadly it's the lib/ move that's bugged